libpod/pod_api.go

```diff
@@ -421,8 +421,9 @@ func (p *Pod) Inspect() (*PodInspect, error) {
 			containerStatus = containerState.String()
 		}
 		pc := PodContainerInfo{
-			ID:    c.ID(),
-			State: containerStatus,
+			ID:      c.ID(),
+			State:   containerStatus,
+			IsPause: c.IsPause(),
```

Comment:
Add PauseContainerID as a separate field in Inspect, not under PodContainerInfo - just print the ID once